src/lib/components/ResFormOpenWater.svelte

```diff
@@ -76,10 +79,11 @@
 			</div>
 		{/if}
 		<div>
-			<select id="formOwTime" {disabled} name="owTime" value={owTime}>
+			<select id="formOwTime" {disabled} name="owTimeManual" value={owTime}>
```

Comment:
what does 'Manual' mean here?

Reply:
i duplicated select so that the real input<owTime> will be used which is hidden. the form doesnt send value of select if disabled.

owTimeManual = user manually inputs
owTime = automatically copies the value of owTimeManual but is hidden just to make form work